config/dev.webpack.config.ts

```diff
@@ -7,7 +7,7 @@ import { updateTsLoaderRule } from './common.webpack.config';
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
     debug: true,
-    https: true,
+    https: false,
```

Comment:
This was part of a test? or is usually better to run in http?

Reply:
No this is required because we have to proxy the dev server from /beta to / in because the historyAPIfallback no does not work on HXR when trying to request the fed-mods.json file. And because of the invalid certificates, we can't use HTTPS without turning off the security checks.